src/js/main.js

```diff
@@ -1,61 +1,64 @@
 (function(){
-
-  var url = "https://refresher.fastlane.tools/duration";
   var startedWithMinutes = null;
-  
+
+  // This number *never* changes
+  var developerSecondsSavedBeforeAnalyticIngester = 22007751774;
+
+  // These numbers are taken from our SQL query
+  var developerSecondsSavedAfterAnalyticIngester = 4292012226;
+  var developerSecondsSavedPerSecond = 560;
+  var timeUpdated = 1499885116;
+
   function reloadThis() {
-    xmlhttp = new XMLHttpRequest();
-    xmlhttp.open("GET", url, true);
-    xmlhttp.send();
-    xmlhttp.onreadystatechange = function() {
-      if (xmlhttp.readyState == 4 && xmlhttp.status == 200) {
-        var results = JSON.parse(xmlhttp.responseText);
-        var seconds = 0;
-        for (var key in results) {
-          seconds += results[key];
-        }
-
-        minutes = seconds / 60; // to minutes
-        if (startedWithMinutes === null) {
-          startedWithMinutes = minutes;
-        }
-        var hours = minutes / 60; // to hours
-
-        var hoursAsArray = parseInt(hours).toString().split('');
-        var outputHtml = '';
-        var hoursOutput = [];
-        hoursAsArray.reverse();
-        
-        for (var i = 0; i < hoursAsArray.length; i++) {
-          if ((i !== 0) && (i % 3 === 0)) {
-            hoursOutput[i] = '<span>'+ hoursAsArray[i] +'</span><span class="comma">,</span>'; 
-          }else{
-            hoursOutput[i] = '<span>'+ hoursAsArray[i] +'</span>';
-          }
-        }
-
-        hoursOutput.reverse();
-        for (var a = 0; a < hoursOutput.length; a++) {
-          outputHtml += hoursOutput[a] ;
-        }
-
-        //document.getElementById("currentCount").innerHTML = parseInt(hours); // .toLocaleString()
-        document.getElementById('currentCount').innerHTML = outputHtml;
-
-        var diff = minutes - startedWithMinutes;
-        if (diff > 1) {
-          // document.getElementById("pageMinutes").innerHTML = diff
-          // document.getElementById("divTransparentMinutes").style.opacity = 1
-        }
-
-        // show everything
-        document.getElementById('timeCounter').style.opacity = 1;
+    // get current time
+    // get difference in seconds from timeUpdated
+    // multiply difference in seconds by developerSecondsSavedPerSecond
+    // add to sum of developerSecondsSavedBefore and After
+
+    var currentEpoch = ((new Date()).getTime()) / 1000;
+
+    var mcDonaldsSeconds = (currentEpoch - timeUpdated) * developerSecondsSavedPerSecond;
```

Comment:
Maybe we shouldn't use that as a variable name, not sure if that might cause problems